generate-assets/src/lib.rs

```diff
@@ -9,12 +18,36 @@ pub struct Asset {
     pub description: String,
     pub order: Option<usize>,
     pub image: Option<String>,
+    pub licenses: Option<Vec<String>>,
+    pub bevy_versions: Option<Vec<String>>,
 
     // this field is not read from the toml file
     #[serde(skip)]
     pub original_path: Option<PathBuf>,
 }
 
+impl Asset {
+    /// Parses a license string separated with OR into a Vec<String>
+    fn set_license(&mut self, license: Option<String>) {
+        if self.licenses.is_some() {
+            return;
+        }
+        if let Some(license) = license {
+            let licenses = license.split("OR").map(|x| x.trim().to_string()).collect();
```

Comment:
should this be `" OR "` to avoid splitting on licenses that happen to contain the characters?

Reply:
Yeah, that's probably a good idea
```suggestion
            let licenses = license.split(" OR ").map(|x| x.trim().to_string()).collect();
```